packages/app-mobile/components/NoteEditor/MarkdownToolbar.tsx

```diff
@@ -465,49 +465,51 @@ const MarkdownToolbar = (props: ToolbarProps) => {
  * This uses react hooks, so should be called in the same places, the same number of times,
  * regardless of state.
  */
-const getStyles = (styleProps: any, theme: any): StyleSheet => {
-	const BUTTON_SIZE = 56;
-
-	return StyleSheet.create({
-		button: {
-			width: BUTTON_SIZE,
-			height: BUTTON_SIZE,
-			alignItems: 'center',
-			justifyContent: 'center',
-			backgroundColor: theme.backgroundColor4,
-			color: theme.color4,
-		},
-		buttonActive: {
-			backgroundColor: theme.backgroundColor3,
-			color: theme.color3,
-		},
-		buttonActiveContent: {
-			color: theme.color3,
-		},
-		text: {
-			fontSize: 22,
-			color: theme.color4,
-		},
-		multiLineIconText: {
-			fontSize: 14,
-			lineHeight: 10,
-			paddingTop: 4,
-			color: theme.color4,
-		},
-		toolbar: {
-			flex: 0,
-			flexDirection: 'row',
-			alignItems: 'baseline',
-			justifyContent: 'center',
-			height: BUTTON_SIZE,
-
-			...styleProps,
-		},
-		toolbarContent: {
-			flexGrow: 1,
-			justifyContent: 'center',
-		},
-	});
+const useStyles = (styleProps: any, theme: any): any => {
```

Comment:
Thanks for making the change. Shouldn't that return a StyleSheet type?

Reply:
If it returns a StyleSheet type, TypeScript prevents us from getting styles from it without a cast (styles.button isn't a property of the StyleSheet type).

Reply:
ok, fair enough. Instead of returning "any", could you try returning nothing? In that case I think type-checking should still be enabled (in that TS may detect the object structure)? In any case when it's possible it's better to return nothing and let auto-type detection do the job.

Reply:
Good idea! I've made that change!